docs/writing-policies/go/07-automate.md

```diff
@@ -1,60 +1,59 @@
 ---
-sidebar_label: "GitHub Action Integration"
-title: ""
+sidebar_label: GitHub Actions
+title: Integrating with GitHub Actions
+description: Integrating with GitHub actions when developing policies for Kubewarden in Go.
+keywords: [kubewarden, kubernetes, github, integration]
+doc-type: [tutorial]
+doc-topic: [kubewarden, writing-policies, golang, github-action-integration]
+doc-persona: [kubewarden-developer]
 ---
 
-# Automations
+## Automation
 
-This section describes how we can use GitHub Actions to automate as many tasks
-as possible.
+This section describes how you can use GitHub Actions to automate tasks.
 
-The scaffolded project already includes all the GitHub actions you need.
-These Actions can be found in the `.github/workflows/ci.yml.template` file;
-rename it to `github/workflows.ci/yml` to enable them.
+The project scaffolding already includes all the GitHub actions you need.
+You can find the Actions in the `.github/workflows/ci.yml.template` file.
+Rename it to `.github/workflows.ci/yml` to enable them.
 
-The same principles can be adapted to use a different CI system.
+You can adapt these principles to use a different CI system.
 
 ## Testing
 
-Automation of the unit tests and of the end-to-end tests is working out of the
-box thanks to the `unit-tests` and `e2e-tests` jobs defined in
-`.github/workflows/ci.yml.template`.
+Automation of the unit tests and of the end-to-end tests works out of the box.
+It uses the `unit-tests` and `e2e-tests` jobs defined in `.github/workflows/ci.yml.template`.
```

Comment:
Actually, the go policy template already came with two files [`test.yml` ](https://github.com/kubewarden/go-policy-template/blob/main/.github/workflows/test.yml)and `release.yml`. There is no need for rename the files. Furthermore, they make use of our github [actions](https://github.com/kubewarden/github-actions/blob/main/.github/workflows/reusable-test-policy-go.yml) that runs the linters and call `make test` and `make e2e-tests`.

Reply:
Adjusted, thanks.